### ISSUE TITLE
Misleading error message for duplicate CLI flags

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [microsoft/TypeScript#43201]
# Bug Report

### 🔎 Search Terms

tsc multiple flag options cli

### 🕗 Version & Regression Information

- This is the behavior in every version I tried, and I reviewed the FAQ for entries about CLI option parsing

### ⏯ Playground Link

N/A

### 💻 Code

Any of:

```ts
tsc -b -b
tsc -b --build
tsc -b --clean --build
```

### 🙁 Actual behavior

```
tsc -b -b      
error TS5072: Unknown build option '-b'.
```

### 🙂 Expected behavior

Something indicating the flag was provided twice, not that it's unknown?

```
tsc -b -b      
error TS5072: Duplicate build option '-b' provided.
```
